testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -594,6 +594,51 @@ public void testRolesAndAllowedOriginsRemovedFromAccessToken() throws Exception
         }
     }
 
+    /**
+     * Chris Nurse 20-Feb-2022
+     * 
+     * PR-4205 serialised single values as an array. This is not desired when the Multivalued options is set to false.
```

Comment:
Consider removing this sentence. Tests should be about the behavior the application should show. Ideally it would reference a spec that has describes the behavior.